examples/demo_threading/images/RLogo.png.license

```diff
@@ -0,0 +1,4 @@
+SPDX-FileCopyrightText: 2022 Klarälvdalens Datakonsult AB, a KDAB Group company <info@kdab.com>
```

Comment:
We need to add a link to this license:
https://creativecommons.org/licenses/by/4.0/

See: https://foundation.rust-lang.org/policies/logo-policy-and-media-guide/

Same in the RLogoLarge license file

Reply:
Changed to `CC-BY-4.0`